recipes/sos-python/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "sos-python" %}
+{% set version = "0.18.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 360dd16db42664d3a609037354bd92c435a246510079621c906be9203338051c
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - sos-notebook >=0.19.4
+
+test:
+  requires:
+    - nose
+    - pytest
+    - selenium
+  imports:
+    - sos_python
+
+about:
+  home: https://github.com/vatlab/sos-python
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
+  summary: "SoS language module for Python2 and Python3"
```

Comment:
not really for python 2 right?

Reply:
This is a language module for [SoS Notebook](vatlab.github.io). It works for both python2 and python3 but I do not want to get into the trouble of testing Python2 support in the recipe (because I will then have to create a python2 environment and install a python2 kernel there).

Reply:
notebooks are strange so maybe you can help me understand the usecase:

As this package stands, could a user use sos-python and a python2 environment together?

Reply:
No. SoS Notebook is based on Python3 and cannot be run in a Python2 environment. It supports Python2 in the sense that you can have on notebook with both SoS, Python3, and Python2 kernels, and SoS allows data exchange between the kernels. `sos-python` module enables such data exchange.

Reply:
got it.